seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,34 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
-        'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
-        ),
-    }
-    return get_json_for_queryset(models, additional_values=additional_values, **kwargs)
+EXPRESSION_OUTLIERS = 'outliers'
+SPLICE_OUTLIERS = 'spliceOutliers'
+
+
+def get_json_for_rna_seq_outliers(filters, significant_only=True, individual_guid=None):
+    filters = {'sample__is_active': True, **filters}
+
+    data_by_individual_gene = defaultdict(lambda: {EXPRESSION_OUTLIERS: {}, SPLICE_OUTLIERS: defaultdict(list)})
+
+    for model, outlier_type in [(RnaSeqOutlier, EXPRESSION_OUTLIERS), (RnaSeqSpliceOutlier, SPLICE_OUTLIERS)]:
+        significant_filter = {f'{model.SIGNIFICANCE_FIELD}__lt': model.SIGNIFICANCE_THRESHOLD}
+        if hasattr(model, 'MAX_SIGNIFICANT_OUTLIER_NUM'):
+            significant_filter['rank__lt'] = model.MAX_SIGNIFICANT_OUTLIER_NUM
+
+        nested_fields = [{'fields': ('sample', 'tissue_type'), 'key': 'tissueType'}]
+        nested_fields += [] if individual_guid else [{'fields': ('sample', 'individual', 'guid'), 'key': 'individualGuid'}]
```

Comment:
So its okay to do it this way, but had you looked at the examples I put in my last comment, you would have seen that you can provide a "value" to the nested_field config that will be used instead of a db query if present without you needing to add custom handling logic, which I think will be neater than what you did here
```
{'fields': ('sample', 'individual', 'guid'), 'key': 'individualGuid', 'value': individual_guid}
```

Reply:
Okay. I didn't know the `value` field is for a default value.